Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReport.cs

```diff
@@ -51,9 +52,32 @@ public override bool SupportRunOnConfig
             get { return true; }
         }
 
+        private string mHTMLReportFolderName;
         [IsSerializedForLocalRepository]
-        public string HTMLReportFolderName { get; set; }
-
+        public string HTMLReportFolderName
+        {
+            get
+            {
+                return mHTMLReportFolderName;
+            }
+            set
+            {
+                mHTMLReportFolderName = value;
+                OnPropertyChanged(nameof(HTMLReportFolderName));
+            }
+        }
+        ValueExpression mValueExpression = null;
```

Comment:
instead of value expression, create a field HTMLFolderNameCalculateda and use it